Formula/png++.rb

```diff
@@ -11,9 +12,55 @@ class Pngxx < Formula
     sha256 "bed85536ebfec14b98b056e874d116527f154eff2b77024aee52fc407d66791f" => :sierra
   end
 
+  depends_on "imagemagick" => :build
   depends_on "libpng"
 
+  # Issues with GNU strerror_r being used because Darwin libc does not define _POSIX_C_SOURCE
+  # Also patches the test script to use imagemagick instead of binary comparison
+  patch :DATA
+
   def install
+    system "make", "check"
```

Comment:
We do not run checks during build (except for security software). Thus this is not needed, and imagemagick is not needed.

Reply:
I can remove it. I just looked at other formulas and saw others were doing it already. It's just not possible to run those tests after installation.

Reply:
Let's remove it, please

Reply:
Done.